spec/lib/web_of_science/xml_parser_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'htmlentities'
+
+describe WebOfScience::XmlParser do
+  let(:wos_encoded_record) { File.read('spec/fixtures/wos_client/wos_encoded_record.html') }
+  let(:wos_record_encoded) { WebOfScience::Record.new(encoded_record: wos_encoded_record) }
+
+  shared_examples 'it has well formed XML' do
+    let(:html_char) { '&lt;' }
```

Comment:
This is only used once.  It doesn't seem to need `let` semantics.

Reply:
OK, removed